src/som/interpreter/Invokable.java

```diff
@@ -46,6 +46,10 @@ public final boolean isAtomic() {
     return isAtomic;
   }
 
+  public ExpressionNode getRoot() {
```

Comment:
`Node.getRootNode()` isn't any good for this?

Reply:
I think Node.getRootNode answers the root node, i.e. the method or primitive, while this method  answers the sequence of expression held by the root node. So I would say no.

Reply:
Done